scripts/make_cppcheck.sh

```diff
@@ -1,13 +1,9 @@
 #!/bin/bash
 
 echo 'make_cppcheck.sh script...'
-echo '#!/bin/bash'
-echo '#!/bin/bash' > copy_cppcheck.sh
 for target in $(make help | grep cppcheck | grep -v '\<all_cppcheck\>' | awk '{print $2}'); do
     mkdir $target
     make $target > $target/cppcheck-result.log 2> $target/cppcheck-result.xml
     sed -i -e "s@/opt/$1/@@g" $target/cppcheck-result.xml
```

Comment:
Is this script going to be used for anything other than kpsr-core? If not, I see no benefit in passing the folder $1 as arg.

Reply:
Yes, it will be used for kpsr-robotics too.